### PR TITLE
Issue-315: Review API and deployment readme

### PR DESCRIPTION
**Pull request by chinmayic**
Review complete deployment readme (readme.md) and API doc readme (docs/readme.md).

### CODE REVIEW

File: svc-aggregation/README.md [hunk 1]
```diff
@@ -638,8 +657,8 @@ curl -i GET \
    "HostName":"10.24.0.4",
    "UserName":"admin",
    "Links":{
-      "ConnectionMethod": {
-         "@odata.id": "/redfish/v1/AggregationService/ConnectionMethods/d172e66c-b4a8-437c-981b-1c07ddfeacaa"
+      "Oem":{
+         "PluginID":"GRF"
```
Comment (jeevan-kamkar) [inline svc-aggregation/README.md]: the response should have the connection method info, not the   plugin id please revert the same

File: svc-aggregation/README.md [hunk 2]
```diff
@@ -699,8 +718,8 @@ curl -i PATCH \
...
```
Comment (jeevan-kamkar) [inline svc-aggregation/README.md]: the response should have the connection method info, not the   plugin id please revert the same

File: svc-aggregation/README.md [hunk 3]
```diff
@@ -504,10 +527,8 @@ Content-Length:491 bytes
 
 ```
 
-
 >**Sample response header \(HTTP 201 status\)**
 
-
 ```
 "cache-control":"no-cache
```
Comment (jeevan-kamkar) [inline svc-aggregation/README.md:546]: add system id in the location  of the response header

**Review by jeevan-kamkar — CHANGES_REQUESTED** [review]
Address the requested changes

File: svc-aggregation/README.md [hunk 4]
```diff
@@ -321,15 +349,16 @@ curl -i POST \
...
 |UserName|String \(required\)<br> |The plugin username.|
 |Password|String \(required\)<br> |The plugin password.|
-|ConnectionMethod|Array (required)|Links to the connection method that are used to communicate with this endpoint: `/redfish/v1/AggregationService/AggregationSources`. To know which connection method to use, do the following:<ul><li>Perform HTTP `GET` on: `/redfish/v1/AggregationService/ConnectionMethods`.<br>You will receive a list of  links to available connection methods.</li><li>Perform HTTP `GET` on each link. Check the value of the `ConnectionMethodVariant` property in the JSON response.</li><li>The `ConnectionMethodVariant` property displays the details of a plugin. Choose a connection method having the details of the plugin of your choice.<br> Example: For GRF plugin, the `ConnectionMethodVariant` property displays the following value:<br>`Compute:BasicAuth:GRF_v1.0.0`</li></ul>|
-
+|PluginID|String \(required\)<br> |The id of the plugin you want to add. Example: GRF \(Generic Redfish Plugin\), ILO, URP |
```
Comment (irappa-pattar) [inline svc-aggregation/README.md]: The request body should have connection method info, not pluginid/PreferredAuthType/PluginType parameter. please revert the changes

File: svc-aggregation/README.md [hunk 5]
```diff
@@ -486,12 +511,10 @@ curl -i -X POST \
...
-|ConnectionMethod|Array (required)|Links to the connection methods that are used to communicate with this endpoint: `/redfish/v1/AggregationService/AggregationSources`. To know which connection method to use, do the following:<ul><li>Perform HTTP `GET` on: `/redfish/v1/AggregationService/ConnectionMethods`.<br>You will receive a list of  links to available connection methods.</li><li>Perform HTTP `GET` on each link. Check the value of the `ConnectionMethodVariant` property in the JSON response.</li><li>The `ConnectionMethodVariant` property displays the details of a plugin. Choose a connection method having the details of the plugin of your choice.<br> Example: For GRF plugin, the `ConnectionMethodVariant` property displays the following value:<br>`Compute:BasicAuth:GRF_v1.0.0`</li></ul>|
-
->**Sample response header \(HTTP 202 status\)**
-
 |Oem\{ PluginID \} \} |String \(required\)<br> |The plugin Id of the plugin.<br> NOTE: Before specifying the plugin Id, ensure that the installed plugin is added in the resource inventory. To know how to add a plugin, see [Adding a Plugin](#adding-a-plugin-as-an-aggregation-source).|
```
Comment (irappa-pattar) [inline svc-aggregation/README.md]: this parameter also not required

File: docs/README.md [hunk 6]
```diff
@@ -1952,7 +1985,10 @@ curl -i POST \
...
 |UserName|String \(required\)<br> |The plugin username.|
 |Password|String \(required\)<br> |The plugin password.|
-|ConnectionMethod|Array (required)|Links to the connection method that are used to communicate with this endpoint: `/redfish/v1/AggregationService/AggregationSources`. To know which connection method to use, do the following:<ul><li>Perform HTTP `GET` on: `/redfish/v1/AggregationService/ConnectionMethods`.<br>You will receive a list of  links to available connection methods.</li><li>Perform HTTP `GET` on each link. Check the value of the `ConnectionMethodVariant` property in the JSON response.</li><li>The `ConnectionMethodVariant` property displays the details of a plugin. Choose a connection method having the details of the plugin of your choice.<br> Example: For GRF plugin, the `ConnectionMethodVariant` property displays the following value:<br>`Compute:BasicAuth:GRF_v1.0.0`</li></ul>|
+|PluginID|String \(required\)<br> |The id of the plugin you want to add. Example: GRF \(Generic Redfish Plugin\), ILO, URP |
```
Comment (irappa-pattar) [inline docs/README.md]: PluginID/PreferredAuthType/PluginType these parameters are not required in request body

File: docs/README.md [hunk 7]
```diff
@@ -2108,7 +2144,8 @@ curl -i -X POST \
...
 |Links \{|Object \(required\)<br> |Links to other resources that are related to this resource.|
-|ConnectionMethod|Array (required)|Links to the connection methods that are used to communicate with this endpoint: `/redfish/v1/AggregationService/AggregationSources`. To know which connection method to use, do the following:<ul><li>Perform HTTP `GET` on: `/redfish/v1/AggregationService/ConnectionMethods`.<br>You will receive a list of  links to available connection methods.</li><li>Perform HTTP `GET` on each link. Check the value of the `ConnectionMethodVariant` property in the JSON response.</li><li>The `ConnectionMethodVariant` property displays the details of a plugin. Choose a connection method having the details of the plugin of your choice.<br> Example: For GRF plugin, the `ConnectionMethodVariant` property displays the following value:<br>`Compute:BasicAuth:GRF_v1.0.0`</li></ul>|
+|Oem\{ PluginID \} \} |String \(required\)<br> |The plugin Id of the plugin.<br> NOTE: Before specifying the plugin Id, ensure that the installed plugin is added in the resource inventory. To know how to add a plugin, see [Adding a Plugin](#adding-a-plugin-as-an-aggregation-source).|
```
Comment (irappa-pattar) [inline docs/README.md]: this is also not required

**Review by irappa-pattar — CHANGES_REQUESTED** [review]

File: svc-aggregation/README.md (hunk 3 above)
Comment (chinmayic) [reply svc-aggregation/README.md:546]: Added.

File: svc-aggregation/README.md (hunk 2 above)
Comment (chinmayic) [reply svc-aggregation/README.md]: Changed.

File: svc-aggregation/README.md (hunk 1 above)
Comment (chinmayic) [reply svc-aggregation/README.md]: Changed.

File: svc-aggregation/README.md (hunk 4 above)
Comment (chinmayic) [reply svc-aggregation/README.md]: Updated with the mentioned changes.

File: svc-aggregation/README.md (hunk 5 above)
Comment (chinmayic) [reply svc-aggregation/README.md]: Removed.

File: docs/README.md (hunk 6 above)
Comment (chinmayic) [reply docs/README.md]: Removed.

File: docs/README.md (hunk 7 above)
Comment (chinmayic) [reply docs/README.md]: Removed.

File: README.md [hunk 8]
```diff
@@ -263,63 +269,46 @@ This section provides a step-by-step procedure for deploying ODIMRA and GRF plug
...
+  **NOTE:**
+  - The resource aggregator configuration files are available at `/etc/odimra_config`.
+  - The GRF configuration files are available at `/etc/grf_plugin_config`.
+  - The URP configuration files are available at `/etc/urp_plugin_config`.
+  - The resource aggregator API service runs on the default port 45000.
+  - The GRF plugin API service runs on the default port 45001.
+  - The URP API service runs on the default port 45003.
+  - The resource aggregator logs are available at `/var/log/odimra`.
+  - The GRF plugin logs are available at `/var/log/GRF_PLUGIN`.
+  - The URP logs are available at `/var/log/URP_PLUGIN`.
```
Comment (skhan-anjum) [inline README.md]: URP logs are not at this location..we can check logs with docker command 'docker logs build_urp_1'

**Review by skhan-anjum — CHANGES_REQUESTED** [review]
Address the changes

File: README.md (hunk 8 above)
Comment (chinmayic) [reply README.md]: Updated this line.

**Review by Bharath-KKB — COMMENTED** [review]
Looks OK . Let the team comment. Can we see the full files somewhere ?

**Comment by a-ajith** [comment]
In Task, the example response is incorrect. The payload field is supposed to have the request payload details, but the example has the response payload.

https://github.com/chinmayic/ODIM/tree/sprint-32-odim-315/docs#tasks

**Comment by skhan-anjum** [comment]
Heading for computer system chassis in sample response body has to be bold like it is for other headings
https://github.com/chinmayic/ODIM/tree/sprint-32-odim-315/docs#single-chassis

File: docs/README.md [hunk 9]
```diff
@@ -272,9 +272,7 @@ Use the following URL in all HTTP requests that you send to the resource aggrega
     have changed the default port in the `odim_config.json` file, use that as the port.
```
Comment (Bharath-KKB) [inline docs/README.md]: remove `` or replace with ''

File: docs/README.md [hunk 10]
```diff
@@ -272,9 +272,7 @@ Use the following URL in all HTTP requests that you send to the resource aggrega
...
```
Comment (Bharath-KKB) [inline docs/README.md:296]: same as above

File: docs/README.md [hunk 11]
```diff
@@ -5894,7 +5892,7 @@ Refer to [Resetting Servers](#resetting-servers) to know about `ResetType.`
 |**URI** |`/redfish/v1/Systems/{ComputerSystemId}` |
```
Comment (Bharath-KKB) [inline docs/README.md:5933]: replace `` with ""

File: docs/README.md [hunk 12]
```diff
@@ -5894,7 +5892,7 @@ Refer to [Resetting Servers](#resetting-servers) to know about `ResetType.`
...
 |**Description** |This action changes the boot settings of a specific system such as boot source override target, boot order, and more.<br>**IMPORTANT**<br><ul><li>Ensure that the system is powered off before changing the boot order.</li><li>Power on the system once the operation is successful. The changes will be seen in the system only after a successful reset.</li></ul><br> To know how to power off, power on, or restart a system, see [Resetting a computer system](#resetting-a-computer-system).|
 |**Returns** |Message Id of the actual message in the JSON response body. To get the complete message, look up the specified registry file \(registry file name can be obtained by concatenating `RegistryPrefix` and version number present in the Message Id\). See [Message Registries](#message-registries). For example,`MessageId` in the sample response body is `Base.1.0.Success`. The registry to look up is `Base.1.0`.<br> |
-|**Response code** |`200 OK` |
+|**Response code** |`200 OK`|
```
Comment (Bharath-KKB) [inline docs/README.md:5936]: remove `` or replace with ''

**Review by Bharath-KKB — CHANGES_REQUESTED** [review]
address comments

**Review by jeevan-kamkar — COMMENTED** [review]
https://github.com/chinmayic/ODIM/pull/11/files#diff-fff4bd4f819daada2ec0df000ae9ab119bc7045a65922542887db3aaf2b39cafR622
 change the `OdataType:    "#AggregationSourceCollection.v1_0_0.AggregationSourceCollection",` to `OdataType:    "#AggregationSourceCollection.AggregationSourceCollection",`

**Review by jeevan-kamkar — APPROVED** [review]

**Review by shivacharanms — APPROVED** [review]

**Review by skhan-anjum — APPROVED** [review]

**Review by bharath-b-hpe — APPROVED** [review]

File: docs/README.md (hunk 11 above)
Comment (Bharath-KKB) [reply docs/README.md:5933]: as discussed this is internal to markup so this should not be actioned.

**Review by Bharath-KKB — APPROVED** [review]

**Review by irappa-pattar — APPROVED** [review]
